src/features/licenses/components/LicensesPage/index.jsx

```diff
@@ -48,6 +49,7 @@ const LicensesPage = () => {
         license: form.license.id,
         institution: form.license.institution,
         catalogs: form.license.catalogs || [],
+        licenseType: form.license.licenseType,
```

Comment:
@AuraAlba What do you think about adding the license type also when editing the license?